store/deposit-dates.js

```diff
@@ -43,7 +40,7 @@ class DepositDates extends Store {
 
     if (total === 0) return 0
     const level = (compliant / total) * 100
-    return Math.round(level * 100) / 100
+    return [total, Math.round(level * 100) / 100]
```

Comment:
I would create 3 separate computed properties: `totalCount`, `nonCompliantCount`, `complianceLevel`. It's more clear than `complianceLevel[0]`